src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -496,6 +496,19 @@ Please change the value to an assembly-qualifed type name which inherits from '{
   <data name="XA1035" xml:space="preserve">
     <value>The 'BundleAssemblies' property is deprecated and it has no effect on the application build. Equivalent functionality is implemented by the 'AndroidUseAssemblyStore' and 'AndroidEnableAssemblyCompression' properties.</value>
     <comment>The following are literal names and should not be translated: 'BundleAssemblies', 'AndroidUseAssemblyStore', 'AndroidEnableAssemblyCompression'</comment>
+  </data>
+	<data name="XA1036" xml:space="preserve">
+    <value>Microsoft.Android {0} does not support a deployment target of '{1}' (the minimum is '{2}'). Please increase the $(SupportedOSPlatformVersion) property value in your project file.</value>
+    <comment>The following are literal names and should not be translated: Microsoft.Android, $(SupportedOSPlatformVersion)
+{0} - The version of the SDK
+{1} - The value of the property
+{2} - The minimum SupportedOSPlatformVersion supported by this version of the SDK</comment>
+  </data>
+  <data name="XA1037" xml:space="preserve">
+    <value>AndroidManifest.xml //uses-sdk/@android:minSdkVersion '{0}' does not match the $(SupportedOSPlatformVersion) value '{1}' in the project file (if there is no $(SupportedOSPlatformVersion) value in the project file, then a default value has been assumed). Either change the value in the AndroidManifest.xml to match the $(SupportedOSPlatformVersion) value, or remove the value in the AndroidManifest.xml (and add a $(SupportedOSPlatformVersion) value to the project file if it doesn't already exist).</value>
```

Comment:
As per https://github.com/xamarin/xamarin-android/pull/8026/files#r1199275269 , I think this should be a multi-line error message.  *Even if* VS line-wraps error messages -- something I forget -- it should be easier to read if it spans multiple lines.  I would suggest each sentence on a new line.

Reply:
Updated this to put each sentence on a new line.